docs/getting-started.md

```diff
@@ -0,0 +1,344 @@
+```
+
+##### Monitoring pods
+
+Pods are the smallest units in Kubernetes deployments. They can contain one or multiple containers (that need to share resources in order to work together). [Learn more about pods](https://kubernetes.io/docs/concepts/workloads/pods/).
+
+When you click on a pod in the "Workloads -> Pods", you access the pod list. By selecting a pod, you will see the "Details" section that contains information related to the pod,and a "Containers" section that includes the information related to this pod's container(s).
+
+Follow these instructions to access pod and container information:
+
+* To check the status of your deployments in detail, navigate to the "Workloads -> Pods" section located on the left menu. It shows the pod list:
+
+![dashboard-pods](images/dashboard-pods.png)
+
+* Click the pod you'd like to access further details for.
+
+![dashboard-pod-detail](images/dashboard-pod-detail.png)
+
+* As indicated in the image above, you will find a "View logs" link and a "Exec into pod" link at the top right corner. Click either option to review logs for possible errors that might have occurred or to directly access the pod itself.
+
+Alternatively, you can grab the equivalent Pod information using kubectl:
+
+```shell
+$ kubectl get pods
+```
+
+For detailed information around a given pod, replacing POD_NAME
+with the pod you want to inspect:
+
+```shell
+$ kubectl describe pod POD_NAME
+```
+
+For logs around a given pod, replacing POD_NAME
+with the pod you want to grab logs from:
+
+```shell
+$ kubectl logs POD_NAME
+```
+
+For a shell directly into the pod, replacing POD_NAME with the pod you want to exec into:
+
+```shell
+$ kubectl exec --stdin --tty POD_NAME -- /bin/bash
+```
+
+#### Find application credentials
+
+The Timesketch login  credentials are shown in the "Notes" section after installing the application chart:
+
+![notes-txt-secret](images/notes-txt-secret.png)
+
+> NOTE: When deploying the Turbinia Helm chart with the Oauth2 Proxy enabled, you will see an additional secret for the Turbinia Oauth2 config.
+
+Alternatively, to get it from the Kubernetes Dashboard, follow these instructions:
+
+* Navigate to the "Config and Storage -> Secrets" section located on the left menu.
+
+* Click the application for which you wish to obtain the credentials.
+
+* In the "Data" section, click the eye icon to see the password:
+
+![dashboard-secrets](images/dashboard-secrets.png)
+
+### Step 7: Uninstall an application using Helm
+
+To uninstall an application, you need to run the *helm uninstall* command. Every Kubernetes resource that is tied to that release will be removed.
+
+> TIP: To get the release name, you can run the *helm list* command.
+
+```shell
+    $ helm uninstall my-release
+```
+
+> NOTE: Remember that `my-release` is a placeholder, replace it with the name you have used during the chart installation process.
+
+To delete accompanying the Persistent Volumes, run:
+
+```shell
```

Comment:
this will only delete the pvc, not the pv. I ran into issues doing a uninstall and delete pvc when trying to re-install (it said the pv already existed).

Reply:
Ah good catch, I've updated this to instead be `kubectl delete pvc --all` which will delete all Volumes and added another part to delete an individual PVC. For me it seems like when you delete the individual PVC the PV would then be removed but can you please double check same behavior for you with updated instructions?

Reply:
It works on minikube like you said but failed for me on k3s. As minikube is the standard in this doc let's keep it like that.